SofascoreAcademy/SofascoreAcademy/Controllers/EventsViewController.swift

```diff
@@ -12,11 +12,11 @@ import UIKit
 
 class EventsViewController: UIViewController, BaseViewProtocol {
     
-    var tournaments = TournamentHeaderModel.sampleData
-    var events = EventModel.sampleData
-    
+    private var tournaments = TournamentHeaderModel.sampleData
+    private var events = [EventModel.sampleDataFootball, EventModel.sampleDataBasketball, EventModel.sampleDataAmFootball]
     private let tableView: UITableView = .init()
-    
+    private var tabIndex: Int = UserDefaultsHelper[.tabBarIndex]
```

Comment:
UserDefaultsHelper.tabBarIndex

A tamo trebaš imati
let tabBarIndexKey: String = "tabBarIndexKey"

Slobodno tamo definiraj string keyeve. U Strings fileu bi trebalo biti samo što se vidi na screenu. Ovako si još išao lokalizirati taj key :D što nebi radilo kad bi promjenio drugi jezik jer bi se promjenio i key